tooling/build/scripts/publishing/index.ts

```diff
@@ -313,6 +313,27 @@ function generateSitemapTree(
           : `${pathPrefixWithoutLeadingSlash}/${META_PERMALINK}`),
   )?.content?.order
 
+  const pageOrderFromIndex = resources
+    .find(
+      (resource) =>
+        resource.type === "IndexPage" &&
+        resource.fullPermalink ===
+          (pathPrefixWithoutLeadingSlash.length === 0
+            ? INDEX_PAGE_PERMALINK
+            : `${pathPrefixWithoutLeadingSlash}/${INDEX_PAGE_PERMALINK}`),
+    )
+    ?.content?.content?.at(-1)
```

Comment:
suggestion: I think we should use `.find` instead to find for the `childrenPages` block?

Reply:
sounds good; i did this because it's always the last block but i think making the code clearer sounds godo to me too